changelog.md

```diff
@@ -0,0 +1,36 @@
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added 
+
+### Fixed
+
+## [1.1.0] - 2021-04-18
+
+### Added
+
+- Customizable filtering of displayed exams
+- Optionally view password when inserting it on login page
+- Class number in schedule lectures
+- Highlighting most relevant exams
+
+### Changed
+
+- Fixed bus schedule not working (switched API)
+- Fixed issue in timestamp display
+- Add fallback when fetching schedules to deal with API inconsistency
+- Other minor bugs fixed
```

Comment:
Maybe add the fix to not being able to see schedule updates in the schedule page?